website/client/src/components/tasks/task.vue

```diff
@@ -706,6 +777,11 @@
     transition-duration: 0.15s;
     transition-property: border-color, background, color;
     transition-timing-function: ease-in;
+    border: transparent solid 1px;
```

Comment:
I think this is causing habits with a disabled state to lose their border 

<img width="357" alt="Schermata 2020-09-14 alle 16 26 25" src="https://user-images.githubusercontent.com/589911/93098364-10cc0980-f6a7-11ea-804e-b1e028415085.png">
 because it's overriding this rule 
<img width="303" alt="Schermata 2020-09-14 alle 16 26 06" src="https://user-images.githubusercontent.com/589911/93098344-0c9fec00-f6a7-11ea-8b2a-7873e757dd73.png">

if it becomes too difficult to make it work, I'm fine with using `!important` on the disabled state rule

Reply:
@paglias I ended up having to use !important :/ 

Here's the new screen capture, changes were made in https://github.com/HabitRPG/habitica/pull/12363/files#diff-6385a285d7711bdd02a0ea7d765961a9

![disabled-habit-styling](https://user-images.githubusercontent.com/8680662/93418475-58959100-f85f-11ea-991c-d895ee9a6074.gif)

Reply:
thanks! @Tressley this is headed up to delta